ludwig/encoders/h3_encoders.py

```diff
@@ -411,7 +411,7 @@ def __init__(
             num_layers=num_layers,
             bidirectional=bidirectional,
             use_bias=use_bias,
-            dropout=dropout,
+            dropout=recurrent_dropout,
```

Comment:
I think that we're missing `recurrent_dropout` in the docstring for this class. Could you add that in?